README.md

```diff
@@ -163,8 +163,9 @@ Development
 A local checkout of the frameworks repo can be shared with locally-running services (i.e. frontend applications)
 as follows:
 
-- from this repo, run `node_modules/bower/bin/bower link`
-- from each app, run `node_modules/bower/bin/bower link digitalmarketplace-frameworks`
+- install bower globally (if you didn't already): `npm install -g bower`
+- from this repo, run `bower link`
+- from each app, run `bower link digitalmarketplace-frameworks`
```

Comment:
Is it worth also adding instructions how to UNLINK once you're done.  In my experience having things linked locally causes more problems than it's worth, so knowing how to undo this damage is useful.

Also, what's the thinking behind global rather than local bower? (I have no idea about this at all)

Also (I know not part of this PR), is it worth removing the bit about "(NB this step also required before deployment to PaaS via `cf push`)"?  The Frameworks repo seems like a really weird place to have that information, and it's not something we should be encouraging anyone to do much.

Reply:
Apparently, Bower is supposed to be installed globally. See Caroline's comment: https://github.com/alphagov/digitalmarketplace-frameworks/issues/412#issuecomment-291151394

_Hopefully_ we'll get rid of bower quite soon!

Reply:
Specifically global rather than local because this repo doesn't have a local place to install bower (which is why the instructions were wrong in the first place).

I reckon people can probably look up the docs to unlink, I'm not sure adding something here would be enough of a reminder (i.e. that this could be the cause) when things go wrong.